docs/modules/ROOT/pages/how-tos/storage/change-luks-key.adoc

```diff
@@ -0,0 +1,62 @@
+= Change LUKS Key on an Encrypted Rook Managed Ceph OSD
+
+This page describes how to change the LUKS encryption key of an OSD backing storage when using Rook to manage the Ceph cluster.
+
+== Information Gathering
+
+. Figure out the LUKS device which we'll be working with by looking into the Pod spec to figure out the name of the PVC used
++
+----
+kubectl -n rook-ceph describe pod rook-ceph-osd-${OSDID}-... | grep ClaimName
+----
++
+The path on the node will then be:
++
+----
+/var/lib/rook/rook-ceph/${PVCNAME}/ceph-${OSDID}/block-tmp
```

Comment:
This will depend on the namespace in which the Ceph cluster is provisioned.

```suggestion
/var/lib/rook/${CEPHCLUSTER_NAMESPACE}/${PVCNAME}/ceph-${OSDID}/block-tmp
```